src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../App.css';
+import ButtonPanel from './buttonPanel';
+import Display from './display';
+
+function App() {
+  return (
+    <>
+      <Display result="0" />
+      <ButtonPanel />
+    </>
+  );
+}
```

Comment:
## Use ES6 Arrow function

- [ ] kindly implement this function component with the recommended with an ES6 arrow function.
```suggestion
const  App = () => {
  return (
    <>
      <Display result="0" />
      <ButtonPanel />
    </>
  );
}
```